index.js

```diff
@@ -22,7 +30,18 @@ function getPercentageScore(score) {}
  * @param {object} score2 - An object representing another score of an assignment. Has the same shape as `score1`.
  * @returns {number} A rounded integer, representing the highest percentage scored among the two given scores.
  */
-function getHighestOfTwoScores(score1, score2) {}
+function getHighestOfTwoScores(score1, score2) {
+  let highScore1 = 0;
+  let highScore2 = 0;
+    highScore1 = (score1.received / score1.max) * 100;
+    highScore2 = (score2.received / score2.max) * 100;
+
+  if (Math.round(highScore1) > Math.round(highScore2)){
+    return Math.round(highScore1);
+  } else {
+    return Math.round(highScore2);
+  }
+}
```

Comment:
What could Math.max do for you here? (Spoiler alert: make your code much shorter!)